lib/src/results.dart

```diff
@@ -23,22 +23,21 @@ import 'dart:ffi';
 import 'collections.dart';
 import 'native/realm_core.dart';
 import 'realm_class.dart';
-import 'realm_object.dart' show RealmObjectMetadata;
+import 'realm_object.dart' show RealmObjectMetadata, RealmEntityInternal;
```

Comment:
That's not right. We should not be exposing `Internal` classes.

Reply:
Not sure how that got there - will investigate.

Reply:
Rereading this, we're not exposing the internal class, we're importing it.

Reply:
I removed the `show ...` clause as it seemed redundant.

Reply:
oh I misread that line.